python/ray/autoscaler/aws/example-full.yaml

```diff
@@ -118,6 +118,11 @@ cluster_synced_files: []
 # should sync to the worker node continuously
 file_mounts_sync_continuously: False
 
+# Metadata that can be used to pass custom arguments.
```

Comment:
Please remove these from the example files, I don't think they add much and only clutter them more.